README.md

```diff
@@ -63,178 +72,366 @@ bambuAnnotations <- prepareAnnotations(gtf.file)
+ 
+### Modulating the sensitivity of discovery (pre and post analysis)
 
-You can also use precalculated annotations.
+When doing transcript discovery there is a balance between sensitivity (the number of real novel transcripts that are detected) and the precision (how many of the novel transcripts are real). To control this balance bambu uses the novel discovery rate (NDR) as the main parameter. The NDR threshold approximates the proportion of novel candidates output by bambu, relative to the number of known transcripts it found. I.E an NDR of 0.1 would mean that 10% of all transcripts passing the threshold are classified as novel. 
 
-If you plan to run ***bambu*** more frequently, we recommend to save the bambuAnnotations object.
+If you are using a genome where you expect a high amount of novel transcripts, a higher NDR is recommended so that these novel transcripts are not missed. Conversely if you are using a well annotated genome, we recommend a lower NDR threshold to reduce the presence of false positives. By default the NDR threshold is automatically chosen for the user based on predicted level of annotation completeness when compared to the default model trained on human reference annotations (Hg38). For more information see [Training a model on another species/dataset and applying it](#Training-a-model-on-another-speciesdataset-and-applying-it)
 
-The bambuAnnotation object can be calculated from a *.gtf* file:
+To manually select an NDR value, use the NDR argument in bambu:
 
 ```rscript
-annotations <- prepareAnnotation(gtf.file)
+se.NDR_0.3 <- bambu(reads = test.bam, annotations = annotations, genome = fa.file, NDR = 0.3)
 ```
-
-From *TxDb* object
+Alternatively transcript discovery can be run without thresholds, producing a GRangesList annotation object with all transcripts scored with its NDR score. Note that this means turning quant = FALSE in running bambu (refer to “Transcript discovery only” section). The annotations can be filtered by their NDR score (see example below), read count and gene read proportion between the discovery and quantification steps or used for other types of analysis. 
 
 ```rscript
-annotations <- prepareAnnotations(txdb)
+newAnnotations <- bambu(reads = test.bam, annotations = annotations, genome = fa.file, NDR = 1, quant = FALSE)
+annotations.filtered <- newAnnotations[(!is.na(mcols(newAnnotations)$txNDR) & mcols(newAnnotations)$txNDR<0.1) | is.na(mcols(newAnnotations)$txNDR)]
+se.NDR_1 <- bambu(reads = test.bam, annotations = annotations.filtered, genome = fa.file, NDR = 1, discovery = FALSE)
+
+Additionally there are other thresholds that advanced users can access through opt.discovery when running *bambu* (see arguments).
 ```
+### Output
+bambu returns a SummarizedExperiment object which can be accessed as follows:
```

Comment:
It looks nicer if you bold 
assays(se), rowRanges(se), rowData(se), assays(se)$counts , assays(se)$CPM, assays(se)$fullLengthCounts, assays(se)$uniqueCounts